cpu/load_balancer.py

```diff
@@ -161,6 +166,8 @@ def tearDown(self):
         1. Restoring the system with turning on all the core's and smt on.
         2. Killing the stress-ng workload
         """
+        process.run("ps aux | grep '[m]pstat' | grep -v grep | awk \
+                '{print $2}' | xargs kill -9", shell=True)
```

Comment:
Fails when mpstat process in not running in the system.
Happens when for some reason `test()` returned before setting up `mpstat`
<details>
<summary> run log </summary>
$ avocado run load_balancer.py
JOB ID     : 8ee7e8eca94e0b007d2b0246c663f23795c81c49
JOB LOG    : /home/user/avocado/job-results/job-2024-01-09T01.19-8ee7e8e/job.log
 (1/1) load_balancer.py:load_balancer.test: STARTED
 (1/1) load_balancer.py:load_balancer.test:  ERROR: Command 'ps aux | grep '[m]pstat' | grep -v grep | awk                 '{print $2}' | xargs kill -9' failed.\nstdout: b''\nstderr: b'kill: not enough arguments\n'\nadditional_info: None (1.29 s)
RESULTS    : PASS 0 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 2.98 s

Test summary:
1-load_balancer.py:load_balancer.test: ERROR
</details>